docs/src/architecture/state.md

```diff
@@ -13,26 +12,26 @@ The Miden Node(s) maintain three databases to describe the state:
 
 ## State components
 
-These databases are represented by authenticated data structures, such that we can easily prove that items were added to or removed from a database, and a commitment to the database would be very small.
+These databases are represented by authenticated data structures, which allows us to quickly show that objects were added to or withdrawn from a database, and a commitment to the database is minimal.
 
-### Account database
-Current account states are recorded in a Sparse Merkle tree which maps account IDs to account hashes, where the account hash is computed as 
+### Account DB
```

Comment:
Maybe  you can also add here that the Account DB datastructure is a [tiered sparse Merkle tree](https://0xpolygonmiden.github.io/miden-base/crypto-primitives/tsmt.html) as well. And with this I am unsure which words to capitalize. Is it Tiered Sparse Merkle Tree or tiered sparse Merkle tree?

Reply:
We usually prefer Tiered Sparse Merkle Tree (SMT). This has been the standard on zkEVM docs as well.

Reply:
ok, better to be consistent